bundle/manifests/vault-config-operator.clusterserviceversion.yaml

```diff
@@ -235,6 +345,16 @@ spec:
     spec:
       clusterPermissions:
       - rules:
+        - apiGroups:
```

Comment:
not sure what is going on here, but these will be added during the bundle build and should not be here

Reply:
Removed.